todolist/src/actions/index.js

```diff
@@ -0,0 +1,8 @@
+export const ADD_TODO_ACTION = 'ADD_TODO_ACTION'
+
+let todoId=0;
+export const addTodoAction = text => ({
+  type: 'ADD_TODO_ACTION',
```

Comment:
I'd drop the `ACTION` portion of your type names b/c thru naming we know these are actions. Most likely actions will live inside an actions directory as well. Also less typing for us 😀